app/assets/stylesheets/normal.css

```diff
@@ -1,19 +1,25 @@
 h1{
-    font-size: 25px;
+    font-size: 30px;
```

Comment:
Font-related sizes should be in relative units (ems, rems, ...). Might be too late for this project though :crying_cat_face: